subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/inception/BuildSrcApiChangePerformanceTest.groovy

```diff
@@ -77,16 +83,7 @@ class BuildSrcApiChangePerformanceTest extends AbstractCrossVersionPerformanceTe
 
     private static class CreateChangingClassMutator implements BuildMutator {
 
-        private final InvocationSettings settings
```

Comment:
This doesn't look right, but I get the reasons behind it. We should eventually fix this behavior. How about a TODO comment explaining that the code belongs to `beforeScenario`, it just doen't work atm.

Reply:
(I meant to select the entire `beforeScenaro` -> constructor conversion code above)

Reply:
I added a comment. Please take another look.